shell/platform/android/io/flutter/embedding/engine/FlutterEngineGroup.java

```diff
@@ -142,20 +143,38 @@ public FlutterEngine createAndRunEngine(@NonNull Options options) {
     DartEntrypoint dartEntrypoint = options.getDartEntrypoint();
     String initialRoute = options.getInitialRoute();
     List<String> dartEntrypointArgs = options.getDartEntrypointArgs();
+    PlatformViewsController platformViewsController = options.getPlatformViewsController();
+    if (platformViewsController == null) platformViewsController = new PlatformViewsController();
```

Comment:
Would it be clearer to replace the above two lines with ternary expressions?

Reply:
Done~